Source/WebKit/Platform/SharedMemory.h

```diff
@@ -70,6 +71,12 @@ class SharedMemoryHandle {
         Win32Handle;
 #endif
 
+    static SharedMemoryHandle create(Type&&, size_t);
```

Comment:
There's no benefit of having create function that returns the object. That is just a normal constructor.

Reply:
Ok you had mentioned having a `create` method for `SharedMemoryHandle` but I can drop this.